metadata.rb

```diff
@@ -11,4 +11,4 @@
 
 source_url 'https://github.com/chef-cookbooks/chef-ingredient'
 issues_url 'https://github.com/chef-cookbooks/chef-ingredient/issues'
-chef_version '>= 12.7' if respond_to?(:chef_version)
+chef_version '>= 13.4' if respond_to?(:chef_version)
```

Comment:
Is there a reason for 13.4 here vs. an earlier 13?

Reply:
habbit, my last company we had issues with chef 13 on windows with a few of it's resources till around 13.4 so we always set that as the minimum target chef-version. I can set it to 13.1